tests/user_restricted/test_user_restricted.py

```diff
@@ -20,9 +20,31 @@ def test_retrieve_patient(self, headers_with_token):
             retrieve[0]["patient"],
             self.headers
         )
-        helpers.check_retrieve_response_body(response, retrieve[0]["response"])
-        helpers.check_response_status_code(response, 200)
         helpers.check_response_headers(response, self.headers)
+        response_body = json.loads(response.text)
+        assert response.status_code == 200
+        assert response_body["id"] == retrieve[0]["patient"]
+        assert response_body["resourceType"] == "Patient"
+        # check the shape of response
+        assert response_body["address"] is not None
+        assert isinstance(response_body["address"], list)
+
+        assert response_body["birthDate"] is not None
+        assert isinstance(response_body["birthDate"], str)
+        assert len(response_body["birthDate"]) > 1
+
+        assert response_body["gender"] is not None
+        assert isinstance(response_body["gender"], str)
+
+        assert response_body["name"] is not None
+        assert isinstance(response_body["name"], list)
+        assert len(response_body["name"]) > 0
+
+        assert len(response_body["identifier"]) > 0
+        assert isinstance(response_body["identifier"], list)
+
+        assert len(response_body["extension"]) > 0
+        assert isinstance(response_body["extension"], list)
 
```

Comment:
Missing check for the "meta" property

Reply:
I did leave that out intentionally since I think that's something that might be likely to change over time. Do you propose just to check if the meta field exists?

Reply:
yeah I'd say just to check if it exists